client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/index.tsx

```diff
@@ -10,6 +12,10 @@ const DesignSetup: Step = ( props ) => {
 		return <AnchorFmDesignPicker { ...props } />;
 	}
 
+	if ( isEnabled( 'signup/design-picker-generated-designs' ) ) {
```

Comment:
Looks like `signup/design-picker-generated-designs` config is true everywhere, is this correct or do you mean a new config for UDP?